Dockerfile-slim.template

```diff
@@ -72,6 +66,13 @@ RUN set -ex \
 	\
 	&& apt-mark auto '.*' > /dev/null \
 	&& apt-mark manual $savedAptMark \
+	&& find /usr/local -type f -executable -not \( -name '*tkinter*' \) -exec ldd '{}' ';' \
```

Comment:
See https://github.com/docker-library/python/issues/122 and https://github.com/docker-library/python/pull/130 (and https://github.com/docker-library/python/pull/283) for background on `tk` / `tkinter`.